### PR TITLE
SuperTrend indicator added

### PR DESCRIPTION
**Pull request by misaghshakeri**
## Summary
Super Trend indicator added

### CODE REVIEW

**Comment by gcarq** [comment]
@misaghshakeri Thanks for creating this PR, but I can't find `super_trend` in the upstream [qtpylib](https://github.com/ranaroussi/qtpylib). We pulled indicators.py file manually to get rid of ton of unneeded features and dependencies. Any specific reason why you put it here? I think a more appropriate place to put it would be `indicator_helpers.py` as it is covered by pytest, flake8 and mypy or make a PR directly to qtpylib. Please also add one or more testcases for that indicator

File: freqtrade/vendor/qtpylib/indicators.py [hunk 1]
```diff
@@ -557,8 +557,71 @@ def stoch(df, window=14, d=3, k=3, fast=False):
...
+
+    # Compute basic upper and lower bands
+    df['basic_ub'] = (df[ohlc[1]] + df[ohlc[2]]) / 2 + multiplier * df["atr"]
+    df['basic_lb'] = (df[ohlc[1]] + df[ohlc[2]]) / 2 - multiplier * df["atr"]
+
+    # Compute final upper and lower bands
+    df['final_ub'] = 0.00
+    df['final_lb'] = 0.00
+    for i in range(period, len(df)):
+        df['final_ub'].iat[i] = df['basic_ub'].iat[i] \
```
Comment (gcarq) [inline freqtrade/vendor/qtpylib/indicators.py]: Please don't use `if expressions` with such complicated conditions, the traditional `if` statement is more readable

**Comment by vertti** [comment]
@gcarq is correct, `indicators.py` is a direct copy and should not be combined with other indicators. `indicator_helpers.py` is probably also not the best place, I had intended it to be helpers for using indicators, like checking if an indicator went up, or crossed another indicator etc.

I think we need a separate indicators.py for us, but I'm also thinking that we should consider using the TA-wrapper that @berlinguyinca has been building. So maybe the most correct place would be to add the super trend there and then include the whole https://github.com/berlinguyinca/technical lib as a dependency. Maybe even consider moving our helpers there?

**Comment by misaghshakeri** [comment]
@gcarq @vertti  thanks, I am going to modify it. so then I do a PR to https://github.com/berlinguyinca/technical ?

**Comment by berlinguyinca** [comment]
@misaghshakeri I think this would be the most appropriate place

**Comment by xmatthias** [comment]
going to close this *here* - but please reopen in https://github.com/berlinguyinca/technical as the indicator does look interresting!

**Comment by hroff-1902** [comment]
first, how do you import indicators?

our samples contain
```
import freqtrade.vendor.qtpylib.indicators as qtpylib
```
i.e. qtpy indicators are imported as qtypylib, not as indicators.

second, as you can see, this PR was closed and super_trend was not added into freqtrade.vendor.qtpylib.indicators (since it's wrong place for it), it was added into technical, import and use it from there...

so I just wonder where are you using it from?

**Comment by mishaker** [comment]
@hroff-1902 actually this implementation is wrong, that's why the PR was closed. it is should be in technical. and also the algorithm should be changed. 
This is merely a POC I done a long time ago.

**Comment by acemi1** [comment]
https://github.com/mishaker/freqtrade/blob/a9f26b26eeb1b4f8811e029dff535e4c3594e512/freqtrade/vendor/qtpylib/indicators.py
I created the indicators.py file with the codes here. And I referenced this file.

however, the codes were incorrect.
The codes in the link below didn't work either.
https://github.com/freqtrade/freqtrade-strategies/issues/30

**Comment by hroff-1902** [comment]
yes, and it should not work, @mishaker explained above why -- the PR was closed

**Comment by acemi1** [comment]
> evet, ve işe yaramaz, mishaker neden yukarıda açıkladı - PR kapatıldı
> 
> asla birleştirilmemiş 2-3 yıllık kod parçalarını almak ve neden mevcut kod tabanı ile çalışmadığını sormak kötü bir fikir ...

You are right.
However, I thought I was making a mistake because I was a novice.
Or I thought someone might notice the error here and update the python code.

Thank you.

**Pull request closed without merging**